kafka-quickstart/src/main/resources/application.properties

```diff
@@ -1,9 +1,21 @@
 # Configure the Kafka sink (we write to it)
-mp.messaging.outgoing.generated-price.connector=smallrye-kafka
-mp.messaging.outgoing.generated-price.topic=prices
-mp.messaging.outgoing.generated-price.value.serializer=org.apache.kafka.common.serialization.IntegerSerializer
+%dev.mp.messaging.outgoing.generated-price.connector=smallrye-kafka
```

Comment:
Any reason for this change?
I would understand if dev and prod configs would be different, but it's not the case here.

Reply:
Oh I see, in `test` you generate only 3 values.

Reply:
Yes. For what I understand here quarkusio/quarkus/issues/2548 the main idea is to detect problems in dependencies. There is no need to have a real istance of kafka. 

I was obblied to duplicate the values in order to avoid configuration of kafka sink for ```test``` profile (but leave it for dev and prod)